lib/chained_job/process.rb

```diff
@@ -34,7 +34,7 @@ def with_hooks
     end
 
     def options
-      @options ||= { job_class: job_instance.class, worker_id: worker_id }
+      @options ||= { job_class: job_instance.class, worker_id: worker_id, args: @args }
```

Comment:
There's an `attr_reader`, let's use that:

```suggestion
      @options ||= { job_class: job_instance.class, worker_id: worker_id, args: args }
```